app/routes/wiki-page.js

```diff
@@ -99,9 +99,8 @@ export default Route.extend(
 				modelParams.page = params.page;
 			}
 
-			return RSVP.resolve(getPageModel(
+			return RSVP.resolve(this.getPageModel(
 				modelParams,
-				fastboot,
 				this.get('wikiVariables.contentNamespaces')
```

Comment:
There is no need to pass this down. But maybe it's easier to reason about if we pass it from here? I'm not sure.